linera-execution/src/test_utils/solidity.rs

```diff
@@ -111,6 +111,176 @@ contract ExampleCounter {
     get_bytecode(&source_code, "ExampleCounter")
 }
 
+pub fn get_evm_call_wasm_example_counter() -> anyhow::Result<Vec<u8>> {
+    let source_code = r#"
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract ExampleCallWasmCounter {
+  bytes32 universal_address;
+  constructor(bytes32 _universal_address) {
+    universal_address = _universal_address;
+  }
+
+
+  struct CounterOperation {
```

Comment:
It would be nice to explain how this code was generated.

Reply:
For the BCS code, the `serde-reflection` was used.  For the serde_json it was done by hand.